python-interface/src/MiniBotScripts/live_audio_sample.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import matplotlib.pyplot as plt #Used for graphing audio tests
+import pyaudio as pa
+import wave
+from time import sleep
+
+#Constants used for sampling audio
+CHUNK = 1024
+FORMAT = pa.paInt16
+CHANNELS = 1
+RATE = 44100 # Must match rate at which mic actually samples sound
+RECORD_TIMEFRAME = 1.0 #Time in seconds
+OUTPUT_FILE = "sample.wav"
+
+#Flag for plotting sound input waves for debugging and implementation purposes
+TESTING_GRAPHS = False
```

Comment:
It is written when taking an audio sample, be I forgot to make it the default variable for sampling or analyzing